course/page/code.py

```diff
@@ -426,6 +426,12 @@ class PythonCodeQuestion(PageBaseWithTitle, PageBaseWithValue):
         based on its :attr:`access_rules` (not the ones of the flow), a warning
         is shown. Setting this attribute to True will silence the warning.
 
+    .. attribute:: max_auto_feedback_points
+
+        Optional, A non-negative :class:`float` value, default to 1,
+        and it can be increased to 10 (`course.constants.MAX_EXTRA_CREDIT_FACTOR`).
+        It indicates the maximum allowed correctness for auto feedback.
+
```

Comment:
Do we really need that? What do you think of the simpler solution I proposed in #448 to not validate the point count in the runner and instead just do this in the app server?

Reply:
I see this

> Ideally, we'd pass it into the docker container via JSON (as, say, `max_correctness`) and compare with that. 

So I guess that's your prefered solution. Thus I've made it more complex.

Reply:
Yeah. I wrote that, and then last night I realize that I was being an idiot. No need to make it so complicated, IMO. Sorry!

Reply:
Nvm, I started out without reading through your later comments. So the work around is remove that validation part in the docker side, and do it in the app server side?

Reply:
You are right, It's better to do that in the app server, I'm closing this and open another PR.